akka-diagnostics/src/main/resources/reference.conf

```diff
@@ -9,6 +9,214 @@
+
+  # By default the checker will be run when the actor system is
+  # started and it will log recommendations at warning log level.
+  # Set this property to "on" if you want to fail the startup of
+  # the actor system when potential configuration issues are found.
+  # Then an `IllegalArgumentException` is thrown from
+  # ActorSystem.apply/create if there are any issues.
+  fail-on-warning = off
+
+
+  # The checks are by default performed in a background thread after this
+  # delay when the actor system has been started. If this property is set
+  # to 0s or if fail-on-warning=on the checks will be run on the caller's
+  # thread, i.e. same thread as is invoking ActorSystem.apply/create.
+  # Running the checks are normally rather quick, but it can still be
+  # good to run them in the background to not delay the startup.
+  async-check-after = 1s
+
+  # You can disable individual checks by adding the key of the check
+  # as described in the warning messages to this list.
+  disabled-checks = []
+
+  # You can confirm that you know the meaning of a specific configuration
+  # setting that is classified as an advanced "power user" setting by adding
+  # the path of the configuration property to this list. When confirmed it
+  # will not warn about that setting any more.
+  confirmed-power-user-settings = []
+
+  # Application specific properties that are not defined by Akka should
+  # be placed outside the "akka" config tree, but if you still have to
+  # define such a configuration property inside "akka" you can confirm
+  # that it is not a typo or misplacement by adding the configuration
+  # path to this list. All properties starting with that path will not
+  # be checked for typos, i.e. you can add the path of a whole section
+  # here to skip everything inside that section.
+  confirmed-typos = []
+}
+#//#config-checker
+
+# Internal settings for the ConfigChecker. Don't change.
+akka.diagnostics.checker {
+
+  # Definition of all properties or sections that are classified as
+  # advanced "power user" settings. You should use 'confirmed-power-user-settings'
+  # instead of modifying this list.
+  power-user-settings = [
+    # akka-actor
+    akka.version,
+    akka.daemonic,
+    akka.actor.unstarted-push-timeout,
+    "akka.actor.router.type-mapping.*",
+    # task-peeking-mode is a power user setting but changed by Play reference-overrides
+    #akka.actor.default-dispatcher.fork-join-executor.task-peeking-mode,
+    akka.actor.default-dispatcher.thread-pool-executor.task-queue-size,
+    akka.actor.default-dispatcher.thread-pool-executor.task-queue-type,
+    akka.actor.default-dispatcher.attempt-teamwork,
+    akka.actor.default-dispatcher.shutdown-timeout,
+    "akka.actor.mailbox.*",
+    akka.actor.serializers.java,
+    akka.actor.serializers.bytes,
+    "akka.scheduler.*",
+    "akka.io.*",
+    akka.diagnostics.checker.power-user-settings,
+    akka.diagnostics.checker.disabled-typo-sections,
+
+    # akka-remote
```

Comment:
> Shall we add to power settings akka.remote.advanced 

@franciscolopezsancho Please create an issue for that. We have to look into it in more detail. Could be that we have to be more specific than marking all advanced.* as power user.

Reply:
added https://github.com/akka/akka-diagnostics/issues/16